index.php

```diff
@@ -0,0 +1,4 @@
+<?php
+    header( 'Location: https://tic-tac-toe.r4ajeti.repl.run' );
+    die; 
+?>
```

Comment:
Why is a PHP file doing here ?

Reply:
Cus i wana deploy this project also in heroku, but for now i removed the php file will add it later